apps/ai-content-generator/src/locations/Dialog.tsx

```diff
@@ -11,24 +12,24 @@ const Dialog = () => {
   const { feature, entryId, isLoading } = useDialogParameters();
 
   if (isLoading) {
-    return <Spinner size="large" />;
+    return null;
```

Comment:
Just switched this to null since the spinner is never visible

Reply:
Does this state need to exist if we just render `null`? or is it accounting for a split second of loading?